crates/cxx-qt-gen/src/generator/naming/property.rs

```diff
@@ -117,30 +116,30 @@ impl QPropertyNames {
     }
 }
 
-pub fn property_name_from_rust_name(ident: Ident) -> Name {
-    // TODO: ParsedQProperty should probably take care of this already and allow the user to set
-    // their own name for C++ if they want to.
-    // REMOVE THIS FN
-    let cxx_name = ident.to_string().to_case(Case::Camel);
-    Name::new(ident).with_cxx_name(cxx_name)
+fn capitalise_first(str: String) -> String {
+    let mut out = "".to_string();
+    if let Some(first) = str.chars().next() {
+        out.push(first.to_ascii_uppercase());
+    } else {
+        // CODECOV_EXCLUDE_START
+        unreachable!("Name cannot be empty!")
+        // CODECOV_EXCLUDE_STOP
+    }
```

Comment:
I would remove this `else` branch and move the `out.push_str` up into the `if let` part.

That way, this function can never fail and will simply return an empty string if the input is an empty string.

Reply:
done